packages/application-generic/src/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -98,17 +98,24 @@ export class GetNovuIntegration {
 
     switch (command.channelType) {
       case ChannelTypeEnum.EMAIL:
-        return this.createNovuEmailIntegration(organization, command, active);
+        return GetNovuIntegration.createNovuEmailIntegration(
+          organization,
+          command.environmentId,
+          active
+        );
```

Comment:
A small question, why do we need to create integrations when we using the get novu integration method? Shouldn't they be created once when the organization is created?

Reply:
yeah the name on that method in GetNovuIntegration is connected to that we "created" them on the fly before. I will rename them so it is clearer now.

Reply:
Ohh okey, makes sense. Thanks for the explanation.